lib/in/create-package-summary.js

```diff
@@ -30,6 +31,15 @@ function createPackageSummary(moduleName, currentState) {
         return false;
     }
 
+    // Ignore specified '--ignore' package globs
+    const ignore = currentState.get('ignore');
+    if (ignore) {
+        const ignoreMatch = Array.isArray(ignore) ? ignore.some(ignoredModule => minimatch(moduleName, ignoredModule)) : minimatch(moduleName, ignore);
```

Comment:
depcheck API accepts `ignoreMatches` array in the options object (see [example](https://github.com/depcheck/depcheck#api)), it is doing the same logic here. I suppose pass the `ignore` variable here to that option is OK.